mesh/mesh.hpp

```diff
@@ -2261,9 +2261,18 @@ class Mesh
        (default) or nonconforming. */
    void EnsureNCMesh(bool simplices_nonconforming = false);
 
+   /// Return a bool indicating whether this mesh is conforming.
    bool Conforming() const { return ncmesh == NULL; }
+   /// Return a bool indicating whether this mesh is nonconforming.
    bool Nonconforming() const { return ncmesh != NULL; }
 
+   /** Designate this mesh for output as "NC mesh v1.1", meaning it is
+       nonconforming with nonuniform refinement spacings. */
+   void SetScaledNCMesh()
+   {
+      ncmesh->usingScaling = true;
```

Comment:
`snake_case` for member variables, `using_scaling`, `reparent_scale`.

Reply:
Done in e5cfc35f87969a015c88f892f8927267bd9ef4c5